resources/views/service/partials/masthead.blade.php

```diff
@@ -1,15 +1,11 @@
 <div class="header">
     @if (!Auth::guest())
         <div class="logout">
-            <ul>
-                <li>
-                    <a href="{{ route('service.logout') }}"
-                       onclick="event.preventDefault();
-           document.getElementById('logout-form').submit();">
-                        Logout
-                    </a>
-                </li>
-            </ul>
+            <form>
+                <button type="submit" value="logout" class="logout-button" onclick="event.preventDefault(); document.getElementById('logout-form').submit();">
+                    Logout
```

Comment:
Copy; "Log out" or "Logout", i'm not sure, but kinda favour two words. What's the common practice?

Reply:
Agreed, log out seems more action like!